code/modules/hydroponics/biogenerator.dm

```diff
@@ -312,6 +312,11 @@
 	for(var/i in 1 to processed_items_per_cycle)
 		var/obj/item/food/food_to_convert = locate(/obj/item/food) in contents
 
+		if(food_to_convert.flags_1 & HOLOGRAM_1)
+			visible_message(span_notice("[food_to_convert] fades away!"))
+			qdel(food_to_convert)
+			continue
+
 		if(!food_to_convert)
```

Comment:
Also please read the rest of the code before jamming checks in, you're going to make biogenerators runtime to hell. `food_to_convert` is nullchecked here.

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/79397/commits/60148faf219a4a0f5f433717420d75ef426e4237